.pre-commit-config.yaml

```diff
@@ -123,6 +111,13 @@ repos:
         language: system
         pass_filenames: false
 
+      - id: prettier
+        name: prettier
+        "types": [text]
+        language: system
+        pass_filenames: true
+        entry: pnpm exec prettier --write --ignore-unknown
```

Comment:
I don't know why those use `bash -c` but doing so prevents the hook from being able to pass filenames, which is critical to avoid prettier checking every single file in the repo during commit, rather than just those that have changed. The former takes _ages_ and only needs to happen when running with `--all-files` as with `ov just lint` (which is used in CI).